teacher.rb

```diff
@@ -0,0 +1,11 @@
+require './person'
+class Teacher < Person
+  def initialize(name, parent_permission, age, specialization)
```

Comment:
- [ ] Please give `name` a default value!
`name = "Unknown"`

Reply:
Thank you a lot for this guidance @Rustamxon7.
just inquiring, wouldn't the parent class emphasize this on Initialization though?